code/modules/jobs/job_types/garrison/forestguard.dm

```diff
@@ -137,10 +136,10 @@
 
 /datum/outfit/job/forestguard/reaver/pre_equip(mob/living/carbon/human/H)
 	..()
-	neck = /obj/item/clothing/neck/chaincoif
-	beltl = /obj/item/weapon/mace/steel/morningstar
+	neck = /obj/item/clothing/neck/coif/cloth
+	beltl = /obj/item/weapon/mace
```

Comment:
```suggestion
```
They already have a large axe.
they don't need a mace an axe and a wood cutter's axe